src/operator/nn/mkldnn/mkldnn_act.cc

```diff
@@ -43,13 +43,9 @@ namespace mxnet {
 namespace op {
 
 bool SupportMKLDNNAct(const ActivationParam& param) {
-  // We only enable ReLU for now. It seems other activations have some precision
-  // problems.
-  return param.act_type == activation::kReLU;
-#if 0
+  return param.act_type == activation::kReLU
       || param.act_type == activation::kSigmoid
       || param.act_type == activation::kSoftReLU;
```

Comment:
Why tanh is not enabled?

Reply:
@TaoLv  I added support for Tanh and it can pass UT